packages/styled-components/components/menu/themes/default.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+
+/* eslint-disable import/prefer-default-export */
+export const Wrapper = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+`;
+
+// Wrapper around the menu name.
+export const NameWrapper = styled.h3`
+  display: none;
+`;
+
+export const Inner = styled.ul`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  list-style: none;
+`;
+
+export const ItemWrapper = styled.span`
```

Comment:
[The list item element must be a direct child](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) of a UL/OL, this needs to be refactored.